harness/determined/cli/cli.py

```diff
@@ -163,6 +164,7 @@ def render_sequence(sequence: List[str]) -> str:
     + sso.args_description
     + oauth.args_description
     + dev.args_description
+    + config_policies.args_description
```

Comment:
I think we don't want to display help info for config policies until we're ready to release the feature, to prevent customers from accidentally discovering it. Is that possible?

Reply:
Do want the command to still work even if it not displayed under help?

Reply:
+1 to Saloni's question! I was under the impression that these commands wouldn't work 🙈
@azhou-determined do you know if there's a way to "stub" out CLI commands without breaking the python linter?

Reply:
The commands will work to varying degree. The APIs are all stubbed at the moment, so the commands will function but not in a customer-ready way. 

My thought was that, if we added the CLI commands in a non-discoverable way that we could use them to enable testing while we continue to develop features. @salonig23 a feature branch is also ok; I was trying to enable us to merge to main as much as possible for convenience / developer velocity.

Reply:
I will wait to merge this PR after RC cut for the next release - most of the API functions should be implemented by then and I can also work on switching the tests to unit tests intermittently